plugins/node/opentelemetry-instrumentation-fastify/test/instrumentation.test.ts

```diff
@@ -426,5 +428,65 @@ describe('fastify', () => {
         await startServer();
       });
     });
+
+    describe('using requestHook in config', () => {
+      it('calls requestHook provided function when set in config', async () => {
+        const requestHook = (span: Span, info: FastifyRequestInfo) => {
+          span.setAttribute('http.method', info.request.method);
```

Comment:
Can be more idiomatic to use `SemanticAttributes.HTTP_METHOD`, as this is not just a random attribute, it's from semantic conventions